### PR TITLE
Updated synthetic time series creation to use ohlc time series objects

### PR DESCRIPTION
**Pull request by anthonymorast**

### CODE REVIEW

File: libs/timeserieslib/SyntheticTimeSeriesCreator.h [hunk 1]
```diff
@@ -6,128 +6,92 @@
...
 
+                std::string timeFrameFilename = getTimeFrameFilename(timeFrameId);
+                PalTimeSeriesCsvWriter<Decimal> csvWriter(timeFrameFilename, *syntheticTimeSeries);
+                csvWriter.writeFile();
+                mTimeSeriesMap.insert(std::make_pair(timeFrameId, syntheticTimeSeries));
+
+                std::cout << timeFrameId << " " << syntheticTimeSeries->getNumEntries() << std::endl;
+            }
+            
+            std::shared_ptr<OHLCTimeSeries<Decimal>> getSyntheticTimeSeries(int timeFrameId)
```
[the remark this answers is not in the document]
Comment (anthonymorast) [reply libs/timeserieslib/SyntheticTimeSeriesCreator.h:58]: That was my mistake, I put it in as a placeholder and forgot to go back and update it.

**Comment by anthonymorast** [comment]
Made code review changes.

**Review by testhound — CHANGES_REQUESTED** [review]
These changes look good, can you update the unit tests to work with the new interface? I don't think the existing unit tests will work any more with this change.

**Comment by anthonymorast** [comment]
Updated the unit test

**Review by testhound — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**